test_wishlists.py

```diff
@@ -19,6 +19,85 @@ def setUp(self):
 	def tearDown(self):
 		wishlists.db=DatabaseEngine()
 
+
+	def test_wishlists(self):
+		resp = self.app.get('/wishlists')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		self.assertTrue(len(resp.data) > 0)
+
+	def test_read_wishlist(self):
+		resp = self.app.get('/wishlists/wl1')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		data = json.loads(resp.data)
+		self.assertEqual(data['user_id'], 'palak')
+
+	def test_read_wishlist_not_found(self):
+		resp = self.app.get('/wishlists/wl2')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+	def test_item(self):
+		resp = self.app.get('/wishlists/wl1/items')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		self.assertTrue(len(resp.data) > 0)
+
+	def test_item_not_found(self):
+		resp = self.app.get('/wishlists/wl2/items')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+	def test_read_wishlist_item(self):
+		resp = self.app.get('/wishlists/wl1/items/item1')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		data = json.loads(resp.data)
+		self.assertEqual(data['description'], 'test item 1')
+
+	def test_read_wishlist_item_item_not_found(self):
+		resp = self.app.get('/wishlists/wl1/items/item2')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+	def test_read_wishlist_item_wishlist_not_found(self):
+		resp = self.app.get('/wishlists/wl2/items/item2')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+	"""
+		Working test case.
+		This is a test case to check whether a wishlist is created or not.
+		POST verb checked here.
+	"""
+	def test_create_wishlist(self):
+		new_wishlist = {'name':'xynazog','user_id':'123'}
+		data = json.dumps(new_wishlist)
+		resp = self.app.post('/wishlists',data=data,content_type='application/json')
+		self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+		new_json = json.loads(resp.data)
+		self.assertEqual(new_json['name'],'xynazog')
+
+	"""
+		Working test case.
+		This is a test case to check whether an item is added to a wishlist or not.
+		POST verb is checked here.
+	"""
+	def test_create_wishlist_item(self):
```

Comment:
I was just checking the number of wishlists created. No need to create one! Sorry.